src/routes/+page.svelte

```diff
@@ -260,7 +265,9 @@
 		style="position: absolute; bottom: 15; left: 15px; background: white; padding: 10px;"
 	  >
 
-	  <p style="font-weight: bold; margin-bottom: 5px;">Show layer:</p>		
+	  <p style="font-weight: bold; margin-bottom: 5px;">Show layer:</p>
+	  <p style="margin-bottom: 5px;">{networkType}</p>
+	  <p style="margin-bottom: 5px;">Zoom level: {getZoomLevel(map)}</p>	
```

Comment:
This will only render once, when the page starts, because that's how Svelte reactivity works. https://svelte.dev/tutorial/reactive-assignments

If zoom was kept in a variable and Svelte saw the update to it, this would be easy. You're in luck, because svelte-maplibre does the work of listening to MapLibre's events and updating a variable for zoom: https://github.com/dimfeld/svelte-maplibre/blob/61f01abf905d25a8046c00f03763b9383a7366e9/src/lib/MapLibre.svelte#L31

So all you need to do is change https://github.com/cruse-bike/cruse/blob/d0d9416ca2f510fcaa70ad15526577e5c9f4751f/src/routes/%2Bpage.svelte#L120. Define `let zoom = 6;` somewhere in this page, do `bind:zoom` when creating the MapLibre component (https://svelte.dev/tutorial/component-bindings), and then in your HTML, just use `{zoom}`. You could do `{#if zoom > 10}<p>x</p>{:else}<p>y</p>{/if}`